src/v2_0/Differ.php

```diff
@@ -158,7 +158,7 @@ public function diffToArray($from, $to, LongestCommonSubsequenceCalculator $lcs
         }
 
         if ($this->detectUnmatchedLineEndings($diff)) {
-            \array_unshift($diff, ["#Warning: Strings contain different line endings!\n", 3]);
```

Comment:
@keradus same here, not sure why this change is needed for v3.0?

Reply:
i copy-pasted v2 from original repo as well, that was the change from there